src/server/apis/dash-data/fetch-outgoing-data.ts

```diff
@@ -260,7 +280,17 @@ function getStatus(
     if (reviewsRequestingChanges.length > 0) {
       outgoingStatus = {type: 'PendingChanges'};
     } else if (reviewsApproved.length === reviewersCount) {
-      outgoingStatus = {type: 'PendingMerge'};
+      if (latestCommit && latestCommit.status) {
+        const state = latestCommit.status.state;
+        if (state === 'PENDING') {
+          outgoingStatus = {type: 'StatusChecksPending'};
+        } else if (state === 'ERROR' || state === 'FAILURE') {
+          outgoingStatus = {type: 'StatusChecksFailed'};
+        }
+      }
+      if (outgoingStatus.type === 'UnknownStatus') {
+        outgoingStatus = {type: 'PendingMerge'};
+      }
```

Comment:
What about when status checks are passing?

Reply:
the two remaining states are no states and success states, both should result in PendingMerge for the dashboard state.